index.bs

```diff
@@ -4031,37 +4031,59 @@ WebAuthn [=[RP]=].
 
 : Client extension input
 :: The Boolean value `true` to indicate that this extension is requested by the [=[RP]=].
-    <xmp class="idl">
-    partial dictionary AuthenticationExtensionsClientInputs {
-      boolean loc;
-    };
-    </xmp>
 
 : Client extension processing
 :: None, except creating the authenticator extension input from the client extension input.
 
 : Client extension output
-:: Returns a JavaScript object that encodes the location information in the authenticator extension output as a {{Coordinates}} value,
-    as defined by [[Geolocation-API]].
-    <xmp class="idl">
-    partial dictionary AuthenticationExtensionsClientOutputs {
-      Coordinates loc;
-    };
-    </xmp>
+:: Returns a JSON object that encodes the location information in the authenticator extension output as a Coordinates value,
+    as defined by [The W3C Geolocation API Specification](https://dev.w3.org/geo/api/spec-source.html#coordinates_interface).
```

Comment:
This addition is unnecessary, as the client extension output description already covers this.